tests/com/twitter/intellij/pants/integration/OSSPantsCompileActionsTest.java

```diff
@@ -86,19 +87,50 @@ public void testRebuildAction() throws Throwable {
 
   public void testCompileAllTargetsInModuleAction() throws Throwable {
     doImport("testprojects/src/java/org/pantsbuild/testproject/junit");
-    Module module = getModule("testprojects_src_java_org_pantsbuild_testproject_junit_testscope_common_sources");
-    PantsCompileAllTargetsInModuleAction compileAllTargetsInModuleAction =
-      new PantsCompileAllTargetsInModuleAction(Optional.of(module));
-    Set<String> targetAddresses = compileAllTargetsInModuleAction
-      .getTargets(getPantsActionEvent(), myProject)
-      .collect(Collectors.toSet());
-    Set<String> expectedTargets = new HashSet<>(Arrays.asList(
+    Module testscopeModule = getModule("testprojects_src_java_org_pantsbuild_testproject_junit_testscope_common_sources");
+    Module mixedModule = getModule("testprojects_src_java_org_pantsbuild_testproject_junit_mixed_tests_org_pantsbuild_tmp_tests_tests");
+    PantsCompileAllTargetsInModuleAction action = new PantsCompileAllTargetsInModuleAction(Optional.empty());
+    assertFalse(action.module.isPresent());
+
+    VirtualFile vf = PantsUtil.findFileRelativeToBuildRoot(myProject, "testprojects/src/java/org/pantsbuild/testproject/junit/mixed/tests/org/pantsbuild/tmp/tests/AllTests.java").get();
+    Module mockModule = PantsUtil.getModuleForFile(vf, myProject).get();
+    assertEquals(mockModule, mixedModule);
```

Comment:
nit:

expected value is on the left http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(double, double)

same for other assertions

Reply:
Fixed in 436e514.